doc/sphinx/docs/algorithms/mbh.rst

```diff
@@ -0,0 +1,8 @@
+.. cpp_mbh
+
+Monotonic Basin Hopping (MBH) - Generalized
+===========================================================
+
+.. doxygenclass:: pagmo::mbh
+   :project: PaGMOreborn
```

Comment:
This is wrong (project is ``pagmo`` now) and unnecessary anyway as we set the default project in the sphinx config file. It can go.